README.md

```diff
@@ -14,7 +14,7 @@ Running the latest scikit-learn test suite with Intel(R) Extension for Scikit-le
 
 ## 👀 Follow us on Medium
 
-We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-software/tagged/machine-learning) to learn tips and tricks for more efficient data analysis the help of Intel(R) Extension for Scikit-learn. Here are our latest blogs:
+We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-software/tagged/machine-learning) to learn with the help of Intel(R) Extension for Scikit-learn. Here are our latest blogs:
```

Comment:
the sentence doesn't make much sense :)

learn what?

Reply:
applied